File: app/src/main/java/lpiemam/com/apppokecards/RetrofitPoke.java
package lpiemam.com.apppokecards;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.DELETE;
import retrofit2.http.GET;
import retrofit2.http.PUT;
import retrofit2.http.Query;

public interface RetrofitPoke {
    @GET("ApiPokeCards/users.php")
    Call<List<UserInfo>> getUserInfo(@Query("id") int idUser);

    @PUT("/user/info")
    Call<UserInfo> updateUserInfo(
            @Body UserInfo userInfo
    );

    @DELETE("/user")
    Call<Void> deleteUser();

}

File: app/src/main/java/lpiemam/com/apppokecards/RetrofitPoke.kt
package lpiemam.com.apppokecards

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.DELETE
import retrofit2.http.GET
import retrofit2.http.PUT
import retrofit2.http.Query

interface RetrofitPoke {
    @GET("ApiPokeCards/users.php")
    fun getUserInfo(@Query("id") idUser: Int): Call<List<UserInfo>>

    @PUT("/user/info")
    fun updateUserInfo(
            @Body userInfo: UserInfo
    ): Call<UserInfo>

    @DELETE("/user")
    fun deleteUser(): Call<Void>

}